client/packages/invoices/src/Returns/CustomerDetailView/AppBarButtons.tsx

```diff
@@ -59,7 +59,8 @@ export const AppBarButtonsComponent: FC<AppBarButtonProps> = ({
           onClick={() => onAddItem()}
         />
         <ReportSelector
-          context={ReportContext.InboundReturn}
+          context={ReportContext.InboundShipment}
+          subContext="CustomerReturn"
```

Comment:
Using subcontext here since Inbound_return context doesn't exist in report in OG. Same goes for outbound